-------------------------------------------------------------------------------
Test set: TestReceive
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.001 sec <<< FAILURE!
TestReceive.testRec()  Time elapsed: 0 sec  <<< FAILURE!
java.net.BindException: Address already in use: bind
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:550)
	at java.base/sun.nio.ch.DatagramChannelImpl.bindInternal(DatagramChannelImpl.java:1189)
	at java.base/sun.nio.ch.DatagramChannelImpl.bind(DatagramChannelImpl.java:1160)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.bind(DatagramSocketAdaptor.java:107)
	at java.base/java.net.DatagramSocket.createDelegate(DatagramSocket.java:1131)
	at java.base/java.net.DatagramSocket.<init>(DatagramSocket.java:197)
	at java.base/java.net.DatagramSocket.<init>(DatagramSocket.java:256)
	at TestReceive.testRec(TestReceive.java:41)

